website/docs/reference/resource-configs/clickhouse-configs.md

```diff
@@ -186,11 +188,32 @@ dbt snapshots allow a record to be made of changes to a mutable model over time.
 
 </File>
 
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+<File name='snapshots/<model_name>.sql'>
+
+```jinja
+{{
+   config(
+     schema = "<schema_name>",
+     unique_key = "<column-name>",
+     strategy = "<strategy>",
+     updated_at = "<unpdated_at_column-name>",
+   )
+}}
+```
+
+</File>
+
+</VersionBlock>
+
 #### Snapshot Configuration
 
 | Option          | Description                                                                                                               | Required?                                                                            |
```

Comment:
I don't think we need this table? It seems duplicative of the other table here -> https://docs.getdbt.com/docs/build/snapshots#snapshot-configurations